apps/rock-the-steps/src/app/stage-select/stage-select.component.scss

```diff
@@ -85,5 +88,5 @@ ion-icon.is-completed {
   width: 100%;
   color: var(--rts-basic-white);
   text-align: center;
-  bottom: -0.0625;

```

Comment:
On iOS this strikes out and was not influencing, as a result the footer was not displaying at all.